plugins/wporg-5ftf/includes/pledge.php

```diff
@@ -129,3 +131,51 @@ function create_new_pledge( $name ) {
 
 	return wp_insert_post( $args, true );
 }
+
+/**
+ * Filter query for archive & search pages to ensure we're only showing the expected data.
+ *
+ * @param WP_Query $query The WP_Query instance (passed by reference).
+ * @return void
+ */
+function filter_query( $query ) {
+	if ( is_admin() || ! $query->is_main_query() ) {
+		return;
+	}
+
+	// Set up meta queries to include the "valid pledge" check, added to both search and any pledge requests.
+	$meta_queries = (array) $query->get( 'meta_query' );
+	$meta_queries[] = array(
+		'key' => META_PREFIX . 'org-number-employees',
```

Comment:
There will be a different name for this meta value, since it's going to be based on the number of confirmed contributors attached to the pledge, rather than the number of employees reported by the organization.

The working meta key I'm using right now is `META_PREFIX . 'pledge-confirmed-contributors'`, so I would say go with that here (and elsewhere in this function) for now.

Reply:
OK, I've updated the key name.